department-of-veterans-affairs/va.gov-cms
# [WIP] VACMS-2763: create new video entities from Media list - Videos component

## Description
See #2763 

## Testing done
Visual / Behat

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/92030108-927d7800-ed34-11ea-8b08-5e83b298bff5.png)
![image](https://user-images.githubusercontent.com/2404547/92030123-96a99580-ed34-11ea-9709-bcae1f4e4c7b.png)

## QA steps
- As an admin go to `/node/add/media_list_videos` and visually verify that you see a video upload option. (Will require @oksana-c 's feedback)
- Add a youtube url and confirm save is successful
